lib/pltr/v2/reducers/ui.js

```diff
@@ -106,6 +107,15 @@ export default function ui(state = defaultUI, action) {
     case SET_TIMELINE_FILTER:
       return Object.assign({}, state, { timelineFilter: action.filter })
 
+    case SET_OUTLINE_FILTER:
+      if (!action.filter) filter = null
+      else if (Array.isArray(state.outlineFilter) && state.outlineFilter.includes(action.filter))
+        filter = state.outlineFilter.filter((item) => item !== action.filter)
+      else if (!Array.isArray(state.outlineFilter) || !state.outlineFilter) filter = [action.filter]
```

Comment:
`!state.outlineFilter` is redundant here.
We can instead just write:
```
else if (!Array.isArray(state.outlineFilter))
```
because it'll already check whether it's `null`, `undefined` or `false`.